### PR TITLE
Fix: Remove 16G hugepage only for Hash MMU

### PR DESCRIPTION
**Pull request by harish-24**
Currently a PowerVM machine has the capability to run on both Hash and Radix MMU, hence remove 16G hugepages only in case of Radix.

Signed-off-by: Harish <harish@linux.ibm.com>

### CODE REVIEW

File: avocado/utils/memory.py
Comment (willianrampazzo) [inline avocado/utils/memory.py:262]: I'm wondering if this statement isn't too generic. It is looking for `Hash` on the whole file. What happens if we have a new processor with `Hash` in the name, and it also supports huge pages?

Comment (harish-24) [reply avocado/utils/memory.py]: @willianrampazzo makes sense, added a more reliable check here. Thanks.

**Review by willianrampazzo — APPROVED** [review]
LGTM, thanks!

**Comment by harish-24** [comment]
@willianrampazzo Please help in merging this, Thanks.

**Comment by willianrampazzo** [comment]
Hi @harish-24, I know it is not mandatory for the Avocado Project, but we have been trying to wait for at least two approvals before merging a PR.

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**
